utils/repository.py

```diff
@@ -7,3 +7,8 @@ def convert_list_to_tuple_string(ids_list):
         else str(tuple(ids_list))
     )
     return result
+
+def create_sql_in_condition(values, field):
```

Comment:
Maybe, it is better to call first the name of the field and then the list
`def create_sql_in_condition(field, values)`
If you have many elements in the list, the name of the field could be lost